server/spec/content_access_spec.rb

```diff
@@ -91,6 +91,36 @@
     }.to raise_exception(RestClient::BadRequest)
   end
 
+  it "should filter content with incorrect architecture" do
```

Comment:
We either need to extend this test or add a new test to verify that when a consumer generates a cert based on their arch, it doesn't negatively impact consumers in the same org+env with different arches.